lib/sensu-plugins-memory-checks/version.rb

```diff
@@ -2,7 +2,7 @@ module SensuPluginsMemoryChecks
   module Version
     MAJOR = 2
     MINOR = 1
-    PATCH = 0
+    PATCH = 1
```

Comment:
Also this is not a patch as it modifies existing behavior, pretty sure the check can work if say you only want to specify just crit thresholds and not warn for example.